wiki/compatibilty.md

```diff
@@ -0,0 +1,21 @@
+<div id="main_content_wrap" class="outer">
+
+<section id="main_content" class="inner">
+
+## Compatibility
+
+The version of Eiffel REMReM Semantics should implement the desired version of [Eiffel REMReM Protocol Interface](https://eiffel-community.github.io/eiffel-remrem-protocol-interface/).
```

Comment:
I'm confused as to what this means. This sentence doesn't give any additional information of which versions of protocol-interface and semantics that are compatible with each other.

Reply:
changed the content